src/sentry/tsdb/snuba.py

```diff
@@ -39,7 +39,9 @@ class SnubaTSDB(BaseTSDB):
     # Single tenant reads Snuba for these models, and writes to DummyTSDB. It reads and writes to Redis for all the
     # other models.
     non_outcomes_query_settings = {
-        TSDBModel.project: SnubaModelQuerySettings(snuba.Dataset.Events, "project_id", None, None),
+        TSDBModel.project: SnubaModelQuerySettings(
+            snuba.Dataset.Events, "project_id", None, [["type", "!=", "transaction"]]
+        ),
```

Comment:
No issue with this but, is it intentional that only this model should exclude transactions? Because soon (well .... this soon is like nuclear fusion, it is always 30 years away since it has been "soon" since october) transactions would not be present in the events dataset anymore, so all of these will exclude transactions.

Reply:
"Soon" we'll be updating the project overview page.